src/launch-store/index.js

```diff
@@ -434,6 +428,14 @@ const LaunchStorePage = ( { onComplete, query } ) => {
 		};
 	} );
 
+	if ( query.status && query.status === 'success' ) {
```

Comment:
> React Hook "useSelect" is called conditionally. React Hooks must be called in the exact same order in every component render.eslint [react-hooks/rules-of-hooks](https://reactjs.org/docs/hooks-rules.html)

Moved this condition to after the `useSelect` call because of the wrong useSelect usage.